<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#fff;
color :#000</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(15,161 ,153 );
</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Conectar Porta Serial</string>
     </attribute>
     <widget class="QPushButton" name="btn_okPortaSerial">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#dddbd8;
color :#000</string>
      </property>
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>309</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lbl_portaSerial">
         <property name="font">
          <font>
           <family>Source Code Pro</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="text">
          <string>Informe o nome da porta:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_serial">
         <property name="styleSheet">
          <string notr="true">background-color:#dddbd8;
color :#000</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="lbl_serial">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_closeSerial">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>140</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#dddbd8;
color :#000</string>
      </property>
      <property name="text">
       <string>Fechar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Cadastrar</string>
     </attribute>
     <widget class="QGroupBox" name="gb_cadastro">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>581</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>323</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lbl_nomeObjeto">
          <property name="font">
           <font>
            <family>Source Code Pro</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Informe o nome do objeto: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_nomeObjeto">
          <property name="font">
           <font/>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#fff;
color :#000</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="lbl_leitura">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>361</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Inserir">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#dddbd8;
color :#000</string>
       </property>
       <property name="text">
        <string>Inserir</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Emprestimos</string>
     </attribute>
     <widget class="QGroupBox" name="gb_abaEmprestimo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>661</width>
        <height>331</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="btn_emprestimo">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>50</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#dddbd8;
color :#000</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_msgEmprestimo">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>521</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QTableWidget" name="tabelaEmprestimo">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>661</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#e7e7e7;
color :#000</string>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>219</number>
       </attribute>
       <column>
        <property name="text">
         <string>Nome</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Item</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Data do Empr√©stimo</string>
        </property>
       </column>
      </widget>
      <widget class="QGroupBox" name="gb_Emprestimo">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>661</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>439</width>
          <height>50</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lbl_nomeEmprestimo">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Nome:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="le_nomeEmprestimo">
           <property name="styleSheet">
            <string notr="true">background-color:#fff;
color :#000</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tabela de Controle</string>
     </attribute>
     <widget class="QTableWidget" name="tabela">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>661</width>
        <height>251</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#e7e7e7;
color :#000</string>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>219</number>
      </attribute>
      <column>
       <property name="text">
        <string>Item</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>C√≥digo do Item</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Disponibilidade</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Hist√≥rico</string>
     </attribute>
     <widget class="QTableWidget" name="tabelaHistorico">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>661</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#e7e7e7;
color :#000</string>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>164</number>
      </attribute>
      <column>
       <property name="text">
        <string>Item</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Data de Retirada</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Devolu√ß√£o</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Buscar</string>
     </attribute>
     <widget class="QLabel" name="lbl_msgBusca">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>441</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>439</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="lbl_NomeBusca">
         <property name="font">
          <font>
           <family>Source Code Pro</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Informe o nome do item:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="le_buscarNome">
         <property name="styleSheet">
          <string notr="true">background-color:#fff;
color :#000</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_buscar">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#dddbd8;
color :#000</string>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabelaBusca">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>641</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#e7e7e7;
color :#000</string>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>212</number>
      </attribute>
      <column>
       <property name="text">
        <string>Item</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>C√≥digo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Disponibilidade</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>474</x>
        <y>290</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#dddbd8;
color :#000</string>
      </property>
      <property name="text">
       <string>Limpar tabela</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNovo_Cadastro">
   <property name="text">
    <string>Novo Cadastro</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
